### DISCUSSION TITLE
Issue in using Volumetric EigenStrain module

### DISCUSSION THREAD

**Discussion by avtarsinghh1991** [idaholab/moose#23304] [Q&A Modules: Solid mechanics] [answered]
Hello Moose Experts, I am trying to use `ComputeVolumetricEigenstrain` as follows for axisymmetric (RZ) problem. 

```
[./volumetric_eigenstrain]
    type = ComputeVolumetricEigenstrain
    volumetric_materials = volumetric_change
    eigenstrain_name = eigenstrain
    args = ''
    block = 'silicon'
[../]

  [./volumetric_change]
    type = GenericFunctionMaterial
    prop_names = volumetric_change
    prop_values = 't'
    block = 'silicon'
  [../]
```

in which t is end time of the simulation which I am using as t=2. However, when I am trying to extract the volume of the mesh block using `VolumePostProcessor`, I am getting V/V_0=2.6621. Can you please help me to understand why this value is more than 2 when my input value is 2. 

I am also calculating the volumetric_strain using following module:

```
  [./volumetric_strain]
    type = RankTwoScalarAux
    scalar_type = VolumetricStrain
    rank_two_tensor = total_strain
    variable = volumetric_strain
  [../]
```

and with this total volumetric strain I am getting is 1.91. I also wonder why this value is less than 2.
Thank you very much.

Best,
Avtar

**Comment by avtarsinghh1991** [comment]
Update:
 I found out that I need to subtract 1 from the volume ratio.
So, 2.6621-1 = 1.6621

But the volumetric strain I am getting is 1.91.
I have tried with fine mesh as well but the changes are at the second digit level.

Best,
Avtar

**Reply by GiudGiud** [reply]
In which number of dimensions is this?
The volume does not scale linearly if the system is expanding in all directions

**Reply by avtarsinghh1991** [reply]
Hello Guillaume,
This is in axis-symmetric problem (RZ-direction).
Best,
Avtar

**Reply by GiudGiud** [reply, marked as answer]
I think you ll need to double check the analytical formula. Expansion is Z causes a linear expansion in volume but in R it s quadratic with a Pi factor in there
